packages/cloud-cognitive/src/components/PageHeader/PageHeader.js

```diff
@@ -103,14 +105,16 @@ export let PageHeader = React.forwardRef(
     // state based on props only
     const hasActionBar = actionBarItems && actionBarItems.length > 0;
     const hasBreadcrumbRow = !!breadcrumbs || !!actionBarItems;
+    const enableBreadcrumbScroll =
+      in_enableBreadcrumbScroll ?? deprecated_disableBreadcrumbScroll ?? false;
```

Comment:
Shouldn't this be:

```suggestion
      in_enableBreadcrumbScroll ?? !deprecated_disableBreadcrumbScroll ?? false;
```

Which also suggests we should have a test for the deprecated prop while it continues to exist, as we do for the new prop?